#! /bin/sh /usr/share/dpatch/dpatch-run
## 100-bugfix-LOGCXX-284.dpatch by  <matthew@localhost>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes build error on AIX with xlc_r

--- src/test/cpp/helpers/datetimedateformattestcase.cpp	2008-03-31 15:33:09.000000000 -0700
+++ src/test/cpp/helpers/datetimedateformattestcase.cpp	2008-07-17 06:49:43.000000000 -0700
@@ -181,7 +181,7 @@
         //  output the using STL
         //
         std::basic_ostringstream<logchar> buffer;
-#if defined(_USEFAC)
+#if defined(_MSC_VER) && _MSC_VER < 1300
          _USEFAC(locale, std::time_put<logchar>)
              .put(buffer, buffer, &date, fmt.c_str(), fmt.c_str() + fmt.length());
 #else
